imjoy/connection/handler.py

```diff
@@ -786,15 +781,14 @@ def exited(result):
         logger.info("Killing all the plugins")
         await kill_all_plugins(engine, sid)
         return {"success": True}
-    else:
-        try:
-            kill_process(logger, int(kwargs["pid"]))
-            return {"success": True}
-        except Exception:  # pylint: disable=broad-except
-            return {
-                "success": False,
-                "error": "Failed to kill plugin process: #" + str(kwargs["pid"]),
-            }
+    try:
```

Comment:
Did these `else` statement cause linter error? I actually prefer having these `else` which allows me to confirm that every condition is properly handled, although there is no difference for the execution.

Reply:
Yes, it's a lint error. Flat is better than nested. We shouldn't use `else` after `return`.